data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,689 @@ mission "Remnant: Learn Sign Follow Up"
+			branch second
+				"retrieved information" == 2
+			`	Ivan then takes a seat in his own cushioned chair. "Welcome to my office. This is miles above what I use to have. After gaining the attention of the Deep government with the discovers we were making, they immediately had my office moved from the dingy complex it use to be at to here. Much of that work I have you the thank for.`
+				goto reason
+			
+			label second
+			`	Ivan then takes a seat in his own cushioned chair. "Welcome to my office. Much better than the old place, isn't it? After gaining the attention of the Deep government with the discovers we were making, they immediately had my office moved from the dingy complex it use to be at to here. Much of that work I have you the thank for.`
+			
+			label reason
+			`	"So. What brings you here, Captain?" he asks.`
+			branch remnant
+				has "deep: did reveal remnant"
+			choice
+				`	"I was wondering what your research on the wormholes and alien creatures has brought up."`
+				`	"Could I get a copy of your research on the wormholes and alien creatures?"`
+					goto "sprites copy"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	"As far as the wormholes go, we're still trying to theorize how the wormholes in this region of space work. A wormhole in the classical sense is some aberration in the fabric of space that allows for instantaneous movement between two points. You can think of it like a tunnel, where you can go one way or the other. But how then does the loop of wormholes that you discovered work? Four different wormholes creating a loop where travel is unidirectional." Ivan turns his chair and glances out at the courtyard. "We aren't yet sure how they could be functioning like this. Anything we come up with seems to violate some law of physics that we're certain is true."`
+			`	Ivan turns back to you. "As for those alien creatures, they're similarly complexing. Humanity has found life on other worlds before, but it has always conformed to our ideas of what life is given our experience on Earth centuries ago. But these creatures are something else entirely. There are no terrestrial planets in the system you found them in suitable for life, so they must have evolved on the gas giants in the system. How life can even begin in such an environment, let alone evolve to be able to travel the vacuum of space, is a mystery to us.`
+			`	"By the way, do you want to take the honors of naming these creatures? My team has some ideas, but we're not particularly convinced by any of them, so I thought I'd ask you."`
+			choice
+				`	"How about 'void sprites'?"`
+					goto sprites
+				`	"How about 'Floaty McFloat Face'?"`
+					goto joke
+				`	"Sorry, I don't have any ideas."`
+			
+			`	"That's unfortunate," Ivan says. "I've been thinking of something in relation to the void of space. I'll come up with something eventually."`
```

Comment:
Don't like the double `something`, but can't think of anything better myself.

Reply:
Yeah, I wasn't a fan of it either but I moved past it instead of getting myself stuck here.
```suggestion
			`	"That's unfortunate," Ivan says. "I've been thinking of a name in relation to the void of space. I'll come up with something eventually."`
```